app/nation/app/Router.js

```diff
@@ -3783,15 +3787,32 @@ $(function() {
                         firstMonth = parseInt(firstDt[1]);
                         firstDt = new Date(firstYear, firstMonth - 1, 01, 00, 00, 00);
                         today = new Date();
-                        var listCommunity;
+                        var listCommunity,div;
```

Comment:
this var is not used anywhere and also it does not look right to have var with html tag name